server/index.js

```diff
@@ -69,6 +122,71 @@ app.post('/api/uploads', uploadsMiddleware, (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/uploads/ratings/sign-up', uploadsMiddleware, (req, res, next) => {
+  const { ratedWho } = req.body;
+  if (!ratedWho) {
+    throw new ClientError(400, 'Who is being rated is a required field');
+  }
+  const sql = `
+    insert into "ratings" ("whoRated", "ratedWho", "rating")
+
+    SELECT 1 as "whoRated", "ratedWho", 5 as "rating"
+    from
+    (SELECT "username",
+      "accountId" as "ratedWho"
+    from "accounts"
+    where "username" = $1) as "a"
+    returning *;
+  `;
+  const params = [ratedWho];
+  db.query(sql, params)
+    .then(result => {
+      res.json(result.rows[0]);
+    })
+    .catch(err => next(err));
+});
+
+app.post('/api/uploads/ratings', uploadsMiddleware, (req, res, next) => {
```

Comment:
Something weird is going on here, but I think I just figured it out. You are using `uploadsMiddleware` because your `fetch()` requests are `FormData`, instead, just have the client send you JSON 

Also, can we rework the code so that we don't need two ratings POST routes?